README.md

```diff
@@ -244,3 +244,6 @@ docker push ${TAG}
 ```
 
 Instructions on how to run the Import Adapter Docker image locally are available on the [wiki](https://github.com/GoogleCloudPlatform/healthcare-dicom-dicomweb-adapter/wiki/Running-Docker-image-locally).
+
+### Data protection toolkit deployment 
```

Comment:
Please capitalize the title "Data Protection Toolkit Deployment"

Reply:
done